test/Constraints/generics.swift

```diff
@@ -52,9 +52,9 @@ func generic_metatypes<T : SomeProtocol>(_ x: T)
 }
 
 // Inferring a variable's type from a call to a generic.
-struct Pair<T, U> { } // expected-note 5 {{'T' declared as parameter to type 'Pair'}}
+struct Pair<T, U> { } // expected-note 4 {{'T' declared as parameter to type 'Pair'}} expected-note {{'U' declared as parameter to type 'Pair'}}
```

Comment:
It's not clear to me what's going on with the notes here and below. Are there more/less and why?